README.md

```
@@ -41,29 +1,155 @@
+
+
+<br/><br/>
+
+
+
+## 3. 기술 스택 & 도구
+
+
+**개발 환경**
+
+<img src="https://img.shields.io/badge/vite-%23646CFF.svg?style=for-the-badge&logo=vite&logoColor=white"/> <img src="https://img.shields.io/badge/NPM-%23CB3837.svg?style=for-the-badge&logo=npm&logoColor=white"/>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)